deploy/legacy/manifests/controller/helm/retina/values.yaml

```diff
@@ -253,7 +263,20 @@ metrics:
     tlsConfig: {}
     ## @param metrics.serviceMonitor.relabelings [array] Prometheus relabeling rules to apply to samples before scraping
     ##
-    relabelings: []
+    relabelings: 
```

Comment:
I guess you added the config here as the relabeling is going to be shared between the operator and agent in future work part of #847 right?

Reply:
Yes exactly, then all service monitors would be aligned with what we currently have in the additionalScrapeConfig.